packages/node_modules/@webex/internal-plugin-mercury/README.md

```diff
@@ -39,18 +39,22 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
 
 ```js
 const webex = require(`webex`);
+const url = require('url');
 const HttpsProxyAgent = require('https-proxy-agent');
 
-let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
-
-webex.init({
-	config: {
-	  defaultMercuryOptions: {
-		agent: httpsProxyAgent
-	  },
-	 ...
-	}
-});
+// init webex
+let config = {
+  credentials: process.env.TOKEN
+  //...and anything else you need
+};
+let proxyUrl = process.env.http_proxy || null;
+if (proxyUrl) {
+  let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+  config.defaultMercuryOptions = {
+    agent: httpsProxyAgent
+  };
+}
```

Comment:
a new line after this if statement bracket could help improve readability.

Reply:
OK